gsa/src/web/pages/reports/alertactions.js

```diff
@@ -205,6 +210,7 @@ class AlertActions extends React.Component {
 
 AlertActions.propTypes = {
   alerts: PropTypes.array,
+  capabilities: PropTypes.array,
```

Comment:
We have a Capabilities proptype. Please use that one. Also, I guess this should be required, as l.172 would crash if caps are missing.
```suggestion
  capabilities: PropTypes.capabilities.isRequired,
```